src/3rdparty/webkit/Source/WebCore/platform/qt/PlatformScreenQt.cpp

```diff
@@ -39,9 +39,38 @@
 #include "QWebPageClient.h"
 #include <QApplication>
 #include <QDesktopWidget>
+#include <stdlib.h>
 
 namespace WebCore {
 
+static unsigned long qt_get_screen_width() {
+  char *width = getenv("QT_SCREEN_WIDTH");
+  unsigned long screen_width = 1366;
+
+  if (width != NULL) {
+      unsigned long screen_width_tmp = atol(width);
```

Comment:
Use [`QString::toInt`](http://doc.qt.io/qt-4.8/qstring.html#toInt), as that does error checking as well.